### PR TITLE
Adding a new test for FIB in pytest

### PR DESCRIPTION
**Pull request by arlakshm**
Adding a test for FIB in pytest.

Signed-off-by: Arvindsrinivasan Lakshminarasimhan <arlakshm@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?
Convert the ansible fib test to py test
#### How did you verify/test it?
Verify on the test run on existing platform
#### Any platform specific information?
None
#### Supported testbed topology if it's a new test case?
This test is support in T1 and T0 topologies

### CODE REVIEW

**Comment by prsunny** [comment]
@arlakshm , as i understand, this is converting the existing ansible test to pytest right? The title mentions as adding new test.. Just clarifying

**Comment by arlakshm** [comment]
@prsunny, This is new test in pytest, so I marked it as new test.

File: tests/test_fib.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
+from datetime import datetime 
+
+@pytest.mark.parametrize("ipv4, ipv6, mtu", [pytest.param(True, True, 1514)])
+def test_fib(ansible_adhoc, testbed, duthost, ptfhost,ipv4, ipv6, mtu):
+
+    t1_topologies =  ['t1', 't1-lag', 't1-64-lag', 't1-64-lag-clet']
+    t0_topologies =  ['t0',  't0-52', 't0-56', 't0-64', 't0-64-32', 't0-116']
```
Comment (yxieca) [inline tests/test_fib.py]: I am wondering if you could make this more generic?

Having these 2 above list meaning that if we define a new topology in the future. This test code needs to be changed too. Previously we only have to change the testcase.yml.

File: tests/fib/fib.j2 [hunk 2]
```diff
@@ -0,0 +1,95 @@
+{# defualt route#}
+{% if testbed_type == 't1' %}
+0.0.0.0/0 {% for ifname, v in minigraph_neighbors.iteritems() %}
+{% if "T2" in v.name %}{{ '[%d]' % minigraph_port_indices[ifname]}}{% if not loop.last %} {% endif %}{% endif %}{% endfor %}
+
+::/0 {% for ifname, v in minigraph_neighbors.iteritems() %}
+{% if "T2" in v.name %}{{ '[%d]' % minigraph_port_indices[ifname]}}{% if not loop.last %} {% endif %}{% endif %}{% endfor %}
+
+{% elif testbed_type == 't0' or testbed_type == 't0-52'or testbed_type == 't0-64' or testbed_type == 't1-lag' or testbed_type == 't0-64-32' %}
```
Comment (lguohan) [inline tests/fib/fib.j2]: it does not really work for t0 topology now, as the tor_number has been removed from t0 topology file. https://github.com/Azure/sonic-mgmt/pull/1233

I think we need to revisit the fib test later.

Comment (arlakshm) [reply tests/fib/fib.j2]: I used the same fib.j2, which is used by the ansible playbook. 
Should add a check to run this test only for T1 for now  ?

File: tests/test_fib.py (hunk 1 above)
Comment (lguohan) [reply tests/test_fib.py]: good concern, this needs to work together with the new fib fixture. suggest to address in a separate pr.

**Pull request merged**